src/lib/djerba/extract/r_script_wrapper.py

```diff
@@ -255,6 +256,16 @@ def preprocess_gep(self, gep_path):
                 writer.writerow(row)
         return out_path
 
+    def tab_file_is_empty(self, mavis_path):
+        """
+        Checks if .tab file provided is empty.
+        Returns False is not empty, returns True if empty.
+        """
+        with open(mavis_path, "rt") as file:
+            num_lines = file.readlines()
```

Comment:
Confusing since this isn't a number, it's an array holding the contents of the file. Instead, could have: `num_lines = len(file.readlines())` followed by `if num_lines > 1:`.